language/sygus.tex

```diff
@@ -1524,6 +1549,41 @@ \section{Synthesis Solver Output}%
 was unable to find a solution, which does not necessarily imply that
 the conjecture is infeasible.
 
+\item
+A (successful) well-formed response to $\paren{\optsynthkwd\mbox{ }\paren{t_1 \ldots t_j}\mbox{ } A_1 \ldots A_k}$
+is a tuple of terms followed list of commands, enclosed in parentheses, of the form:
+\[
+\begin{array}{l}
+\paren{\\
+\paren{c_1 \ldots c_j}\\
+\paren{\fundefcmdkwd\mbox{ }f_{1}\mbox{ }X_{1}\mbox{ }\sigma_{1}\mbox{ }t_{1}}\\
+\ldots\\
+\paren{\fundefcmdkwd\mbox{ }f_{n}\mbox{ }X_{n}\mbox{ }\sigma_{n}\mbox{ }t_{n}}\\
+}
+\end{array}
+\]
+To be a well-formed response, 
+$c_1, \ldots, c_j$ must be values of the same type as $t_1, \ldots, t_j$,
+and the remaining commands must be a well-formed response based on the criteria
+for $\checksynthkwd$ above.
+A response of the above form is correct if
+the specified functions satisfy the syntactic restrictions given by
+the current state.
+Additionally, the solution must satisfy
+the semantic specification for optimization queries, which
+implies that both the semantic specification is satisfied
+and that the values $c_1, \ldots, c_j$
+are consistent valuations of terms $t_1, \ldots, t_j$
+under the current solution for functions to synthesize $f_1, \ldots, f_n$.
+For details, see \cref{sec:logical-semantics}.
```

Comment:
We do not talk about the attributes $A_1, \ldots, A_k$ here. Please add some explanation for them as well.